site/pom.xml

```diff
@@ -14,7 +14,7 @@
 
 	<properties>
 		<!-- Mars convention -->
-		<publishingPath>mars/snapshots/builds/${JOB_NAME}/${BUILD_ID}-B${BUILD_NUMBER}/all/repo/</publishingPath>
+		<publishingPath>mars/snapshots/builds/${JOB_NAME}/${BUILD_TIMESTAMP}-B${BUILD_NUMBER}/all/repo/</publishingPath>
 		<!-- Luna convention: TODO JBIDE-18772: remove this when master build only works for Mars -->
 		<publishingPath>builds/staging/${JOB_NAME}/all/repo/</publishingPath>
```

Comment:
Oh, so it does actually work as defined in the jenkins job - the mars url contained an old update site, this is the most current:
http://download.jboss.org/jbosstools/neon/snapshots/builds/reddeer_master/
So could you update the PR to say neon, instead of mars? And remove the luna path.
I still don't understand how mvn deploy gets the variable though. But we can remove that definition in the job once this is merged, right?